### DISCUSSION TITLE
Citation link

### DISCUSSION THREAD

**Discussion by J4kson** [Serial-Studio/Serial-Studio#141] [Q&A]
Hello! 
Is there an existing citation link for this project?

**Comment by alex-spataru** [comment]
Uggh, nope 😆, but I think you can cite the project using the website (https://serial-studio.github.io/) or the GitHub repository (https://github.com/serial-studio/serial-studio) for usage. If you end up publishing any paper using this software, let me know—I’d be interested to read it!

**Reply by J4kson** [reply]
Hi Alex,

I published my paper!

Here's the DOI and the full citation:

https://doi.org/10.1016/j.ohx.2023.e00454

Gunther, J., Lengaigne, J., Girard, M., Toupin-Guay, V., Teasdale, J. T., Dubé, M., & Tabiai, I. (2023). A versatile hot melt centrifugal spinning apparatus for thermoplastic microfibres production. HardwareX. Repéré à https://www.hardware-x.com/article/S2468-0672(23)00061-5/fulltext

Hope you'll enjoy it!
Jason
